cc/BUILD

```diff
@@ -4,11 +4,14 @@ load("@io_bazel_rules_docker//docker:docker.bzl", "docker_build")
 load("@package_bundle//file:packages.bzl", "packages")
 
 # An intermediate image for Java and other "mostly statically" compiled languages
-docker_build(
-    name = "cc",
-    base = "//base:base",
+[docker_build(
+    name = "cc" if not mode else mode[1:],
```

Comment:
Does it make more sense to iterate the other way?
```
docker_build(
  name = image + ":debug"
  base = "//" + image
  ...
  directory = "/",
  entrypoint = ["/busybox/sh"],
  env = {"PATH": "/busybox"},
  tars = ["//busybox:busybox.tar"],
...
for image in [
   "cc",
   "java",
   "python",
   ...
```

Reply:
Name can't have a `:` in it.  It also can't have a `/` so things start getting awkward for `java/jetty:debug`.  I also like the symmetry of having the tags match the BUILD labels, but I'm certainly not married to it given another suggestion.

Reply:
Things are further complicated by the lack of symmetry for targets like `java:java8` and `python2.7:python27` :(

Reply:
That make sense.  Just wondering if there was a way we could do it without writing the code for each image.  LGTM

Reply:
I was debating writing a little skylark macro for it, but the quirk in my second comment made that hard enough to stop me.  If it were just `//foo` and `//foo:debug` everywhere I probably would have.  I don't recall the history of why we let these diverge...